js_modules/dagit/packages/ui/src/components/VirtualizedTable.tsx

```diff
@@ -0,0 +1,67 @@
+import {Box, Colors} from '@dagster-io/ui';
```

Comment:
copied this from `core`.

Reply:
Hmm can we remove this file from core now? I think having two copies that export the same things could get confusing

Reply:
I'll do this in a follow up

Reply:
(I have to update a bunch of imports)